packages/client/src/internals/TRPCUntypedClient.ts

```diff
@@ -30,6 +30,7 @@ export interface TRPCSubscriptionObserver<TValue, TError> {
   onStarted: (opts: { context: OperationContext | undefined }) => void;
   onData: (value: TValue) => void;
   onError: (err: TError) => void;
+  onConnecting: () => void;
```

Comment:
Maybe we should have an onState or something that could deprecate all the others